crates/catalog/src/catalogs/embucket/schema.rs

```diff
@@ -92,8 +93,11 @@ impl SchemaProvider for EmbucketSchema {
                 .await
                 .map_err(|e| DataFusionError::External(Box::new(e)))?;
                 let tabular = IcebergTabular::Table(iceberg_table);
+
                 let table_provider: Arc<dyn TableProvider> =
-                    Arc::new(IcebergDataFusionTable::new(tabular, None, None, None));
+                    Arc::new(SnowflakeCaseInsensitiveTable::new(Arc::new(
```

Comment:
Does it apply to all tables? Question if s3 table buckets still work in this case?